content/tutorials/web-rtc-screen-sharing.textile

```diff
@@ -339,7 +339,7 @@ Now, let's go ahead and implement the call, receive and end call features.
 Paste the following code at the end of your @ably-screenshare.js@ file:
 
 
-p(tip). Note: Since Chrome requires an extension to share your screen, we will need to install the temasys chrome plugin "here":https://chrome.google.com/webstore/detail/skylink-WebRTC-tools/ljckddiekopnnjoeaiofddfhgnbdoafc
+p(tip). Note: Since Chrome requires an extension to share your screen, we will need to install the "temasys chrome plugin":https://temasys.io
```

Comment:
> Note: Since Chrome requires an extension to share your screen, we will need to install the "temasys chrome plugin":https://temasys.io

Minor point on style. We try to avoid "we" as then who has the responsibility to install the plugin - the reader or Ably? Sometimes it's not obvious. 

So in this case you can write "you need to install..."

Reply:
OK, I've fixed this. The text is scattered with "we will" because it's come from an external author (I think) but at least this one is resolved and probably one of the most confusing if left unaddressed.

Of course, it's still confusing because there's nothing useful behind the link, but I can't really help that. Sigh. Thanks @tbedford